npm/cmd/start_dataplane.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-container-networking/npm"
+	npmconfig "github.com/Azure/azure-container-networking/npm/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
+)
+
+func newStartNPMDataplaneCmd() *cobra.Command {
+	startNPMDataplaneCmd := &cobra.Command{
+		Use:   "dataplane",
```

Comment:
after dockerfile update, what's the path for reverting out of dataplane/control plane architecture? will `dataplane` or `controlplane` retain classic behavior?

Reply:
As discussed in the call,
- `start` command would start in the current model
- `start controlplane` and `start deamon` would start the controller and daemon in the decomposed mode.